indico/modules/categories/models/categories.py

```diff
@@ -595,6 +605,12 @@ def logo_url(self):
         """Get the HTTP URL of the logo."""
         return url_for('categories.display_logo', self, slug=self.logo_metadata['hash'])
 
+    def get_title(self):
+        return self.title_translations.get(session.lang, self.title)
+
+    def get_description(self):
+        return self.description_translations.get(session.lang, self.description)
+
```

Comment:
Maybe having properties like `i18n_title` and `i18n_description` would be nicer for this?

Reply:
I agree and have changed it.